django_app/docker/web/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN --mount=type=cache,target=$POETRY_CACHE_DIR  poetry install --no-ansi --with
 
 FROM python:3.11-slim-buster as runtime
 
+RUN apt-get update
+RUN apt-get install -y libmagic1
+
```

Comment:
TODO: remove this (it was needed for a file type checking process that I now think would be better served by a call to an external service)